### DISCUSSION TITLE
Trying to use this https://volny.github.io/creative-theme-jekyll/ for my website. But it's not working.

### DISCUSSION THREAD

**Discussion by buildplan** [community/community#124657] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I am a complete novice trying to build a website I am trying to deploy this theme. https://github.com/volny/creative-theme-jekyll

I am following a YouTube video to get me going. https://www.youtube.com/live/q8MzWZRjRoU?si=xT-C7zXehIS42Jjr 

I have followed everything and forked this theme. Now I am stuck where I need to go to the pages option and under Branch need to select main (I have renamed master) and select root but I am getting "Your GitHub Pages site is currently being built from the main branch."

If I go and check _config.yml and make any changes I can't deploy I get this: 
Error:  Logging at level: debug github-pages 231 | Error: (/github/workspace/./_config.yml): found character that cannot start any token while scanning for the next token at line 9 column 19 

My repository is [buildplan.github.io](https://github.com/buildplan/buildplan.github.io)

What am I doing wrong? and how can I fix.

**Comment by kagiyanagi** [comment, marked as answer]
Hey there,

I get that you're stuck trying to deploy the Creative Theme on GitHub Pages. Here's a quick fix for you:

Check _config.yml Syntax:
 Open _config.yml in your repo.
 Look at line 9, column 19. There might be a typo or wrong character.
 It should look something like this:

``` yaml
    title: Your Site Title
    email: your-email@example.com
    description: >-
      Write a brief description here.
    baseurl: ""
    url: "https://buildplan.github.io"
```

Branch Settings:
 Go to your repo on GitHub.
 Click "Settings" > "Pages".
 Make sure the source is set to "main" and the folder to "root".

Commit and Push Changes:
 If you fixed _config.yml, commit and push:
``` bash
        git add _config.yml
        git commit -m "Fix config syntax"
        git push origin main
```
Check the Build Process:
 Go back to "Pages" settings.
 Monitor the build, and it might take a few minutes.

Hope this helps! If you’re still stuck, feel free to share more details. 😊

**Comment by buildplan** [comment]
Hey, thank you so much and much appreciated. I managed to fix it with your help. It seems so obvious now because the error message told me what was the problem. I was trying to add a Twitter profile and GitHub username because the config file had these options already but I think I didn't write it properly so now I have added a # to make it not look for these.

twitter_username: # @P_C_P_S
github_username: # buildplan
